flink-runtime/src/main/java/org/apache/flink/runtime/clusterframework/types/ResourceID.java

```diff
@@ -63,9 +63,7 @@ public final int hashCode() {
 
 	@Override
 	public String toString() {
-		return "ResourceID{" +
-			"resourceId='" + resourceId + '\'' +
-			'}';
+		return resourceId;
```

Comment:
It's mainly for the INFO/DEBUG logs, want to show some simple messages when we have to print ResourceID